src/DynamicMvvm.Abstractions/ViewModel/IDispatcher.cs

```diff
@@ -1,31 +1,23 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Chinook.DynamicMvvm
 {
 	/// <summary>
-	/// A <see cref="IViewModelView"/> represents the view contract of a <see cref="IViewModel"/>.
+	/// An <see cref="IDispatcher"/> allows to execute code on a specific thread.
+	/// This is useful when the <see cref="INotifyPropertyChanged.PropertyChanged"/> event needs to be raised on a specific thread.
 	/// </summary>
-	public interface IViewModelView
+	public interface IDispatcher
 	{
 		/// <summary>
 		/// Gets whether or not the thread has dispatcher access.
 		/// </summary>
 		bool GetHasDispatcherAccess();
 
-		/// <summary>
-		/// Occurs when the view is loaded.
-		/// </summary>
-		event EventHandler Loaded;
```

Comment:
Should we consider this as a breaking changes to remove those events ?

Reply:
I know this was making the VM aware of the view ... I'm just wondering if it was used elsewhere than internally

Reply:
Yes, it's documented as such too :)